src/main.rs

```diff
@@ -94,6 +95,15 @@ async fn main() {
         }
     };
 
+    // Prepare user list
+    match userlist::parse("userlist.json").await {
```

Comment:
You can make this typed. Checkout the example in `config.rs` where we used TOML. The structure of the config file can be typed with a `struct`, so when you parse it you can detect errors & also you won't have to check for them when you use the config elsewhere in the code.